lrpar/README.md

```diff
@@ -96,9 +96,9 @@ use lrlex::lrlex_mod;
 use lrpar::lrpar_mod;
 
 // Using `lrlex_mod!` brings the lexer for `calc.l` into scope.
-lrlex_mod!(calc_l);
+lrlex_mod!("calc.l");
 // Using `lrpar_mod!` brings the lexer for `calc.l` into scope.
```

Comment:
I missed this README in my last PR, so we might as well update this comment here too.

Reply:
This we should probably just fix in this PR.